src/style.css

```diff
@@ -128,4 +128,11 @@ h6 {
   text-decoration: none;
   display: inline-block;
   font-size: 1em;
+  border-radius: 2em;
+  font-weight: bold;
 }
+
+.btn-gotofilms:hover {
```

Comment:
dodaj proszę przejścia między zmianą, żeby nie było to takie gwałtowne
+ żeby kursor się zmieniał na hoverze